docs/plugins/repository-gcs.asciidoc

```diff
@@ -232,7 +232,7 @@ The following settings are supported:
 
     Big files can be broken down into chunks during snapshotting if needed.
     The chunk size can be specified in bytes or by using size value notation,
-    i.e. `1g`, `10m`, `5k`. Defaults to `100m`.
+    i.e. `10m` or `5k`. Defaults to `100m`, which is the maximum permitted.
```

Comment:
I know it was like that before, but it's just wrong. Could you replace `i.e.` by `e.g.` here, please?
Other than that, this looks fine to me.  I added a todo for myself though to look into this limitation (it's coming from our code, not from GCS itself as far as I can tell), it seems unnecessary potentially.